### ISSUE TITLE
Auerswald TFS-Universal Doorbell to HA 

### ISSUE THREAD

**Issue by ballchemist** [swa72/home-assistant#107]
Hi, impressive setup you have made so far.

I am seeking for a moor detailed explanation on the setup for AUERSWALD TFS-Universal Doorbell into the integration to HA.

I have a RASPI5 with HA and a Fritz.Box 7490 - hooked direktly to line 1 the doorbell Auerswald. 

I would love to trigger EVENTS when someone is pushing the doorbell... 

Any more details on your setup, and how to mirror this one would be deeply appreciatet... THanks!

**Comment by swa72** [comment]
That's a bit tricky as it involves the Fritzbox, a cheap USB modem connected to HA (and the phone line of the Fritzbox) and the TFS doorbell connected to a phone line of the Fritzbox

* someone presses the doorbell. TFS calls "**702" 
* "**702" is defined as a group-call inside the Fritzbox, i.e. internally in turn calls "**615#614#610#620#621#2"
* **615, **614, etc. are my DECT phones
* 2 is the modem line (last number in the above group call is the modem)

You can pick any DECT phone and talk to the person at the door and press #9 or 9# (I can't remember which) to buzz the opener.

Now to the modem: the modem notices a ring. That's it. But it is an event nonetheless ;-) So we can trigger automations (on `sensor.phone_modem`, like sending a picture from the camera facing the front door to all local iPhones.

Pretty weird combo but works pretty reliably in my case.

**Comment by swa72** [comment]
Actually I have to add that I've defined a binary sensor that depends on `sensor.phone_modem`

```
    doorbell_pressed:
      friendly_name: "Doorbell was pressed, call initiated"
      value_template: "{{ states('sensor.phone_modem') == 'ring' }}"
```

**Comment by ballchemist** [comment]
Thanks for the pretty detailed answer. I have the similar setting as you already - except the modem for hey Raspberry5… which one did you use?

**Comment by ballchemist** [comment]
Similar to something like this?: https://amzn.eu/d/4phTUq3

**Comment by swa72** [comment]
Yep, according to https://www.home-assistant.io/integrations/modem_callerid/ look for something with a CX93001. I bought mine for cheap off of kleinanzeigen.de (similar to craigslist).

**Comment by swa72** [comment]
No breakthru idea other than try to call another number from the TFS-Universal doorbell (a regular phone) to see if the call initiation works?

**Comment by ballchemist** [comment]
Thanks, seems that there is somewhat a clinche after a ring got accepted, the Modem is not returning into idle... I am currently trying to set idle per default after each ring - but still tricky ...
